src/ImageSharp.Formats.Png/PngEncoderCore.cs

```diff
@@ -495,7 +495,7 @@ private QuantizedImage<TColor> WritePaletteChunk<TColor>(Stream stream, PngHeade
 
             if (this.Quantizer == null)
             {
-                this.Quantizer = new WuQuantizer<TColor>();
+                this.Quantizer = new OctreeQuantizer<TColor>();
```

Comment:
It's doing a lot more work than before.... I added caching in the distance lookup but maybe there is more we can do. 

The Quantizer can be set so dithering could be turned off if they don't want it. I'm making it opt-out since the quality difference is so dramatic.